build.yaml

```diff
@@ -844,9 +841,8 @@ steps:
     script: |
       set -ex
       cd /io/repo/hail
-      chmod 755 ./gradlew
-      time retry ./gradlew --version
-      export SPARK_VERSION="3.0.2" SCALA_VERSION="2.12.10"
+      time retry sh millw --version
+      export SPARK_VERSION="3.0.2" SCALA_VERSION="2.12.13" HAIL_RELEASE_MODE=1
```

Comment:
@danking I only needed to change the scala version in azure, is this something we should manually test?

Reply:
I'm not concerned about a patch version change in Scala